partcad/src/partcad/part.py

```
@@ -20,29 +20,26 @@ class Part(ShapeWithAi):
     url: typing.Optional[str] = None
     count: int = None
 
-    def __init__(self, config: object = {}, shape=None):
-        super().__init__(config)
+    def __init__(self, project_name: str, config: dict = {}, shape=None):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)